test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/RequestTests.cs

```diff
@@ -520,6 +521,53 @@ public void CanUpgradeRequestWithConnectionKeepAliveUpgradeHeader()
             }
         }
 
+        [Theory]
```

Comment:
Because it was already there (I moved it here from `RequestTargetProcessingTests` and adapted as accordingly) 😝 

Will write a unit test.